src/view/actions/directCall.jsx

```diff
@@ -11,41 +11,194 @@
  ****************************************************************************************/
 
 import React from 'react';
-import FullWidthField from '../components/fullWidthField';
+import {
+  ActionButton,
+  TextField,
+  Flex,
+  Well,
+  View
+} from '@adobe/react-spectrum';
+import CloseIcon from '@spectrum-icons/workflow/Close';
+import { FieldArray } from 'redux-form';
+import FullWidthField, {
+  DEFAULT_BLANK_SPACE_PROPS
+} from '../components/fullWidthField';
+import WrappedField from '../components/wrappedField';
+import './directCall.styl';
+
+const FORWARD_SLASH = 0x2215;
+const OPENING_CURLY = 0x007b;
+const CLOSING_CURLY = 0x007d;
+const COLON = 0x003a;
+const SEMICOLON = 0x003b;
+
+const containerMinWidth = 'size-6000';
+
+const DetailObjectRows = ({ fields }) => {
+  return (
+    <>
+      {/* Matches the structure inside FullWidthField */}
+      <Flex alignItems="end" gap="size-100" minWidth={containerMinWidth}>
+        <View flex>
+          <Well marginTop="size-200">
+            <span className="codeLine">
+              <em>
+                {String.fromCharCode(FORWARD_SLASH)}&nbsp;
+                {String.fromCharCode(FORWARD_SLASH)}&nbsp; (optional) - enhance
+                the direct call with this information
+              </em>
+            </span>
+            <span className="codeLine">
+              const detail = {String.fromCharCode(OPENING_CURLY)}
+            </span>
+            {fields.map((detailRow, index) => (
```

Comment:
You could try to use this component https://github.com/adobe/reactor-extension-core/blob/master/src/view/components/multipleItemEditor.jsx here.